dashboard/dashboard.py

```diff
@@ -15,8 +15,8 @@
     {
         'href': 'https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css',
         'rel': 'stylesheet',
-        'integrity': 'sha384-Uu6IeWbM+gzNVXJcM9XV3SohHtmWE+3VGi496jvgX1jyvDTXfdK+rfZc8C1Aehk5',
-        'crossorigin': 'anonymous'
+        #integrity': 'sha384-Uu6IeWbM+gzNVXJcM9XV3SohHtmWE+3VGi496jvgX1jyvDTXfdK+rfZc8C1Aehk5',
+        #'crossorigin': 'anonymous'
```

Comment:
Shouldn't we keep it? And thinking about it, wouldn't be better to add it also to the other css?

Reply:
If you check the console of the browser with this integrity check it raises a lot of errors. Indeed, bootstraps was not working before.

Reply:
I fixed it. Thanks.